README.md

```diff
@@ -98,7 +98,7 @@ running multiple schedulers -- please see the "Scheduler" docs.
 
 Visit the official Airflow website documentation (latest **stable** release) for help with [installing Airflow](https://airflow.apache.org/installation.html), [getting started](https://airflow.apache.org/start.html), or walking through a more complete [tutorial](https://airflow.apache.org/tutorial.html).
 
-> Note: If you're looking for documentation for master branch (latest development branch): you can find it on [ReadTheDocs](https://airflow.readthedocs.io/en/latest/).
+> Note: If you're looking for documentation for master branch (latest development branch): you can find it on [s.apache.org/airflow-docs](https://s.apache.org/airflow-docs/).
```

Comment:
The only risk I see is that this s.a.o link can be overridden, right?

Reply:
I will say more. Only me or INFRA can overwrite this link.